apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/sink/JpaClaimRdaSinkIT.java

```diff
@@ -116,7 +119,7 @@ public void mcsClaim() throws Exception {
     diagCode.setIdrDiagIcdType("T");
     claim.getDiagCodes().add(diagCode);
 
-    int count = sink.writeObject(new RdaChange<>(RdaChange.Type.INSERT, claim));
+    int count = sink.writeObject(new RdaChange<>(RdaChange.Type.INSERT, claim, Instant.now()));
```

Comment:
I had a thought during today's sync call with RDA; most of the time we should always receive the INSERT message for a claim, before any UPDATE message.  However, it's possible that the first message type we get for a claim will be an UPDATE, would this create any problems in processing?

Reply:
Good question.  The `merge()` method is basically an upsert so if the record doesn't exist it will insert it.  It's just not as fast as persist for new records.